src/testlib/phases/GoodWithKey.yml

```diff
@@ -0,0 +1,3 @@
+ForSelfTest:
+  Repeat: {^Parameter: {Name: "Repeat", Default: 1}}
+  Mode: NoException
```

Comment:
I think we should require a `SchemaVersion` on included files and basically assert that the `SchemaVersion` matches the top-level workload file.

Reply:
Done